zmsadmin/js/page/availabilityDay/index.js

```
@@ -164,47 +181,66 @@ class AvailabilityPage extends Component {
-            }).map(cleanupAvailabilityForSave)
-
+            // Format the selected date
+            const selectedDate = formatTimestampDate(this.props.timestamp);
+    
+            // Prepare the data to send
+            const sendAvailability = Object.assign({}, availability);
+    
+            // Clean up temporary fields
+            if (sendAvailability.tempId) {
+                delete sendAvailability.tempId;
+            }
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)